app/assets/stylesheets/common/admin/admin_base.scss

```diff
@@ -1,7 +1,5 @@
 // Styles for /admin section
 
-@import "common/foundation/helpers";
```

Comment:
This import isn't needed in the admin bundle because the main application bundle already includes this import so we don't need to duplicate the same helpers in 2 bundles.